editor/themes/editor_theme_manager.cpp

```diff
@@ -906,6 +906,9 @@ void EditorThemeManager::_populate_standard_styles(const Ref<EditorTheme> &p_the
 			p_theme->set_color("font_disabled_color", "CheckBox", p_config.font_disabled_color);
 			p_theme->set_color("font_outline_color", "CheckBox", p_config.font_outline_color);
 
+			p_theme->set_color("checkbox_checked_color", "CheckBox", p_config.accent_color);
```

Comment:
The editor have different icons for CheckBox, we should force it to be white instead of using accent color.